test/data/dev/deploy_dev_data.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
```

Comment:
```suggestion
set -euo pipefail
```
see https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425